config-templates/redhat/ocs-local/4.6/README.md

```diff
@@ -53,15 +28,11 @@ Run the following commands to create a COS instance, create a set of HMAC creden
     $ ibmcloud resource service-key-create cos-cred-rw Writer --instance-name noobaa-stor --parameters '{ "HMAC": true}'
     ```
 
-3. Create the Kubernetes secret named `ibm-cloud-cos-creds` in the `openshift-storage` namespace that uses your COS HMAC credentials. When you run the command, specify your COS HMAC access key ID and secret access key.
+3. Get the regional public endpoint and the IBM COS location from the UI (Select the region closest to your cluster) or by running this command
     ```
-    $ kubectl -n 'openshift-storage' create secret generic 'ibm-cloud-cos-creds' --type=Opaque --from-literal=IBM_COS_ACCESS_KEY_ID=<access_key_id> --from-literal=IBM_COS_SECRET_ACCESS_KEY=<secret_access_key>
+    $ curl https://control.cloud-object-storage.cloud.ibm.com/v2/endpoints | jq '."service-endpoints"."regional"'
```

Comment:
is there another command that we can give that doesn't require jq? maybe jsonpath instead

```suggestion
    $ curl https://control.cloud-object-storage.cloud.ibm.com/v2/endpoints | jq '."service-endpoints"."regional"'
```

Reply:
@Shirisha-S-Rao-1 @derekpoindexter 
Better point to docs https://cloud.ibm.com/docs/cloud-object-storage?topic=cloud-object-storage-endpoints
https://cloud.ibm.com/docs/cloud-object-storage?topic=cloud-object-storage-classes